CHANGES.rst

```diff
@@ -2,7 +2,7 @@ Changes to be released in next version
 =================================================
 
 ✨ Features
- * 
+ * Data encryption for MXKContactManager and MXKAccountManager using MXKeyProvider (#3866)
```

Comment:
We need to document in the API Changes section that the application must provide a `MXKeyProviderDelegate`.
Else, people will be surprised.